testing/README.md

```diff
@@ -0,0 +1,95 @@
+
+### 2. Create an EKS Cluster 
+The E2E EKS test uses an EKS cluster to deploy the sample apps.
+#### Setup Environment with the Appropriate Roles and Permissions.
+- First, assume Admin role from the test account by running `ada credentials update --account=<AccountID> --role=Admin --provider=isengard --once`
+- Assume the e2e test role by running 
+  - `output=$(aws sts assume-role --role-arn arn:aws:iam::<AccountID>:role/<E2ETestRole> --role-session-name AWSCLI-Session)`
+  - `export AWS_ACCESS_KEY_ID=$(echo $output | jq -r .Credentials.AccessKeyId)`
+  - `export AWS_SECRET_ACCESS_KEY=$(echo $output | jq -r .Credentials.SecretAccessKey)`
+  - `export AWS_SESSION_TOKEN=$(echo $output | jq -r .Credentials.SessionToken)`
+- Run `aws sts get-caller-identity` to check if you are in the correct role
+#### Create a new Cluster
+- Next, create the cluster by running `eksctl create cluster --name <ClusterName> --region us-east-1 --zones us-east-1a,us-east-1b`. This will take around ~10 minutes. 
+#### Install AWS Load Balancer Controller Add-on
+- Finally, install the AWS Load Balancer Controller add-on by running the following commands. Make sure to replace the `<ClusterName>` and `<AccountID>` with the correct value.
+  - `eksctl utils associate-iam-oidc-provider --cluster <ClusterName> --region us-east-1 --approve`
+  - `curl -O https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.4.7/docs/install/iam_policy.json`
+  - `aws iam create-policy --policy-name AWSLoadBalancerControllerIAMPolicy --policy-document file://iam_policy.json --region us-east-1`
+  - `eksctl create iamserviceaccount --cluster=<ClusterName> --namespace=kube-system --name=aws-load-balancer-controller --attach-policy-arn=arn:aws:iam::<AccountID>:policy/AWSLoadBalancerControllerIAMPolicy --region us-east-1 --approve`
+  - `kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.5.4/cert-manager.yaml`
+  - `curl -Lo v2_4_7_full.yaml https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.7/v2_4_7_full.yaml`
+  - `sed -i.bak -e '561,569d' ./v2_4_7_full.yaml`
+  - `sed -i.bak -e 's|your-cluster-name|<ClusterName>|' ./v2_4_7_full.yaml`
+  - `kubectl apply -f v2_4_7_full.yaml`
+  - `curl -Lo v2_4_7_ingclass.yaml https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.7/v2_4_7_ingclass.yaml`
+  - `kubectl apply -f v2_4_7_ingclass.yaml`
+
+### 3. Setting up Environment for EC2 Tests
+#### Create IAM Role for EC2 Instance
+- Login to AWS, go to the IAM console and click on the Roles tab. Click Create role.
+- Choose AWS service, and choose EC2 as the use case. Click Next.
+- Choose AmazonS3ReadOnlyAccess, AWSXrayWriteOnlyAccess, and CloudWatchAgentServerPolicy as the permission. 
+- Type the role name and click "Create role".
```

Comment:
Let's ask them to name their role the same as we do. This would avoid the need for secret updates everywhere and a change in other repos I believe.

Reply:
The CW Agent team has already added the secret for us and we just need to update the variables once this PR is merged. I think this change will make it more flexible and also hide our role name? Lmk

Reply:
Showing role name is not a big deal, and I think this will help reduce the work involved in running the test/having to update the other repo.